### ISSUE TITLE
Unable to build torch-sys

### ISSUE THREAD

**Issue by jxuanli** [guillaume-be/rust-bert#262]
I ran into the problem where when running the conversation.rs example:
```
... (a bunch of warnings followed by)
error occurred: Command "g++.exe" "-O0" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-m64" "-I" "C:\\TempAccess\\rusty_playground\\bert\\target\\debug\\build\\torch-sys-5194cb4590192424\\out\\libtorch\\libtorch\\include" "-I" "C:\\TempAccess\\rusty_playground\\bert\\target\\debug\\build\\torch-sys-5194cb4590192424\\out\\libtorch\\libtorch\\include/torch/csrc/api/include" "-o" "C:\\TempAccess\\rusty_playground\\bert\\target\\debug\\build\\torch-sys-5194cb4590192424\\out\\libtch/torch_api.o" "-c" "libtch/torch_api.cpp" with args "g++.exe" did not execute successfully (status code exit code: 1).
```

My speculation why that might not be working is that my libtorch is `1.12` instead of `1.11`, I am not sure if this is causing the issue thanks!

**Comment by jxuanli** [comment]
I tried with `1.11` still not working, is there a way for me to know exactly what went wrong? printing tons of warning messages doesn't really help

**Comment by guillaume-be** [comment]
Hi @jxuanli ,

Can you please share more information on the platform you are trying to run from, the version of the library (this library and libtorch) and the environment variables you have set up (`LIBTORCH`)?

**Comment by jxuanli** [comment]
Thanks for your reply! I have resolved the issue, see the discussion in https://github.com/LaurentMazare/tch-rs/issues/502

But for future reference, I initially ran my project on windows, and I tried the `Release` version of both `1.11.0` and `1.12.0`. This both gave me the error mentioned in this thread. I was then told to use Linux so I switched to ubuntu. I first tried the `Pre-cxx11 ABI` version of `1.11.0` which gave me the error mentioned in https://github.com/LaurentMazare/tch-rs/issues/502. But I have resolved this by using the `cxx11 ABI` version of `1.11.0`

I would suggest changing the link in Readme.md from `https://download.pytorch.org/libtorch/cu113/libtorch-shared-with-deps-1.11.0%2Bcu113.zip` (the pre-cxx version) to `https://download.pytorch.org/libtorch/cu113/libtorch-cxx11-abi-shared-with-deps-1.11.0%2Bcu113.zip` (the cxx version) as it might be misleading
